fe/src/main/java/org/apache/doris/analysis/AlterClause.java

```diff
@@ -23,6 +23,12 @@
 
 // Alter table clause.
 public abstract class AlterClause implements ParseNode {
+    // if set to true, the corresponding table should be stable before processing this operation on it.
+    protected boolean needTableStable = false;
```

Comment:
I think it's better to set this field default value true. Because if some clause don't set this false, it won't cause big problem, just can not do something.

Reply:
OK, I change it to default true.